src/.clasp.json

```diff
@@ -1 +1 @@
-{"scriptId":"1jJgoGYkwfpbbBudel-_8uNgy5HNRDIw55ussFre7CEcicdLHaN-VNUjw"}
+{"scriptId":"1BGOEacXkSZG5iX_gwDIXWCmqTM8T8jKdHIJznLV21pJ-VROCEs40ZQDc"}
```

Comment:
Move this .clasp.json file to your own folder

Reply:
I updated the branch "issue67-welcome-email", move .clasp.json file to welcomeEmail folder.